src/browser/client.ts

```diff
@@ -244,7 +245,7 @@ export class BrowserClient extends EventEmitter {
   /**
    * @private
    */
-  _getError() {
+  _getError(): HttpError | undefined {
```

Comment:
why would this be `HttpError`? (related to the other request, this error interface shouldn't be public).